examples/thermostat/efr32/BUILD.gn

```diff
@@ -59,6 +60,9 @@ declare_args() {
 
   # Argument to Disable IPv4 for wifi(rs911)
   chip_enable_wifi_ipv4 = false
+
+  # Enable the temperature sensor
+  use_temp_sensor = true
```

Comment:
Should the initialization below be used here, like:

```
use_temp_sensor = (silabs_board =! "BRD2703A") && (silabs_board != "BRD4319A")
```

Reply:
done